{"failedTests":[{"testName":"parallel/test-async-context-frame","timestamp":1725913992449,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win2019-COMPILED_BY-vs2022 #30134","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2019-COMPILED_BY-vs2022/30134/","buildNumber":30134,"builtOn":"test-rackspace-win2019_vs2019-x64-2","nodeVersion":"23.0.0","commitHash":"1a1f9abb8ca673d6f98d30f13855a58dba20d93d","callStack":[{"upstreamBuild":65738,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":74221,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":62200,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54739"}],"tap":{"ok":false,"name":"parallel/test-async-context-frame","id":435,"buffered":false,"tapError":null,"skip":false,"todo":": Fix flaky test","previous":null,"plan":null,"diag":{"severity":"flaky","exitcode":1,"stack":"▶ AsyncContextFrame\n  ✔ async-hooks\\test-async-local-storage-args.js (1543.0749ms)\n  ✔ async-hooks\\test-async-local-storage-async-await.js (1612.9267ms)\n  ✔ async-hooks\\test-async-local-storage-async-functions.js (1837.1867ms)\n  ✔ async-hooks\\test-async-local-storage-dgram.js (1848.2241ms)\n  ✔ async-hooks\\test-async-local-storage-enable-disable.js (1258.5804ms)\n  ✔ async-hooks\\test-async-local-storage-enter-with.js (1361.2738ms)\n  ✖ async-hooks\\test-async-local-storage-errors.js (1923.936ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-errors.js failed with exit code 3221225477\n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:59:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Suite.processPendingSubtests (node:internal/test_runner/test:584:7) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✖ async-hooks\\test-async-local-storage-gcable.js (1393.8725ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-gcable.js failed with exit code 3221225477\n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:59:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Suite.processPendingSubtests (node:internal/test_runner/test:584:7) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-http-agent.js (1713.2437ms)\n  ✔ async-hooks\\test-async-local-storage-http.js (1531.567ms)\n  ✔ async-hooks\\test-async-local-storage-misc-stores.js (1298.9731ms)\n  ✔ async-hooks\\test-async-local-storage-nested.js (1409.2994ms)\n  ✔ async-hooks\\test-async-local-storage-no-mix-contexts.js (1565.262ms)\n  ✔ async-hooks\\test-async-local-storage-promises.js (1149.961ms)\n  ✔ async-hooks\\test-async-local-storage-socket.js (1223.6721ms)\n  ✔ async-hooks\\test-async-local-storage-thenable.js (1118.3846ms)\n  ✔ async-hooks\\test-async-local-storage-tlssocket.js (1330.7677ms)\n  ✔ parallel\\test-async-local-storage-bind.js (1198.0376ms)\n  ✔ parallel\\test-async-local-storage-contexts.js (1159.5144ms)\n  ✔ parallel\\test-async-local-storage-deep-stack.js (1149.0879ms)\n  ✔ parallel\\test-async-local-storage-exit-does-not-leak.js (1129.3552ms)\n  ✔ parallel\\test-async-local-storage-http-multiclients.js (1438.6557ms)\n  ✔ parallel\\test-async-local-storage-snapshot.js (1167.4457ms)\n▶ AsyncContextFrame (32383.2235ms)\nℹ tests 23\nℹ suites 1\nℹ pass 21\nℹ fail 2\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 32398.8959\n\n✖ failing tests:\n\ntest at test\\parallel\\test-async-context-frame.mjs:49:5\n✖ async-hooks\\test-async-local-storage-errors.js (1923.936ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-errors.js failed with exit code 3221225477\n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:59:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:888:9)\n      at async Suite.processPendingSubtests (node:internal/test_runner/test:584:7) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }\n\ntest at test\\parallel\\test-async-context-frame.mjs:49:5\n✖ async-hooks\\test-async-local-storage-gcable.js (1393.8725ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-gcable.js failed with exit code 3221225477\n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:59:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:888:9)\n      at async Suite.processPendingSubtests (node:internal/test_runner/test:584:7) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }"},"time":32946.528,"fullname":"parallel/test-async-context-frame","closingTestPoint":false}}]}